ingestion/src/metadata/ingestion/processor/pii.py

```diff
@@ -0,0 +1,128 @@
+#  Copyright 2021 Collate
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#  http://www.apache.org/licenses/LICENSE-2.0
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+"""
+Processor util to fetch pii sensitive columns
+"""
+import logging
+import traceback
+from enum import Enum
+
+from presidio_analyzer import AnalyzerEngine
+
+from metadata.generated.schema.entity.classification.tag import Tag
+from metadata.generated.schema.entity.data.table import Table, TableData
+from metadata.ingestion.ometa.ometa_api import OpenMetadata
+from metadata.utils import fqn
+
+PII = "PII"
+
+
+class TagType(Enum):
+    SENSITIVE = "Sensitive"
+    NONSENSITIVE = "NonSensitive"
+
+
+class NEREntity(Enum):
+    CREDIT_CARD = TagType.SENSITIVE.value
+    EMAIL_ADDRESS = TagType.SENSITIVE.value
+    IBAN_CODE = TagType.SENSITIVE.value
+    IP_ADDRESS = TagType.NONSENSITIVE.value
+    NRP = TagType.NONSENSITIVE.value
+    LOCATION = TagType.SENSITIVE.value
+    PHONE_NUMBER = TagType.SENSITIVE.value
+    MEDICAL_LICENSE = TagType.SENSITIVE.value
+    US_DRIVER_LICENSE = TagType.SENSITIVE.value
+    DATE_TIME = TagType.NONSENSITIVE.value
+    URL = TagType.SENSITIVE.value
+    US_BANK_NUMBER = TagType.NONSENSITIVE.value
+    US_SSN = TagType.SENSITIVE.value
+    PERSON = TagType.SENSITIVE.value
+    US_PASSPORT = TagType.SENSITIVE.value
+
+
+class NERScanner:
+    """A scanner that uses Spacy NER for entity recognition"""
+
+    def __init__(self, metadata: OpenMetadata):
+        self.metadata = metadata
+        self.text = ""
+        self.analyzer = AnalyzerEngine()
+
+    def get_highest_score_label(self, labels_score):
```

Comment:
Can you add the return type for this function

Reply:
Yes 👍